irohad/torii/processor/impl/query_processor_impl.cpp

```diff
@@ -29,20 +29,20 @@ namespace iroha {
      * @return true if user has needed signatories, false instead
      */
     bool signaturesSubset(
-        const shared_model::interface::SignatureSetType &signatures,
+        const shared_model::interface::SignatureRangeType &signatures,
```

Comment:
You can remove the function at all.

Could you also remove `from_old_model.hpp` include in this file, please?

https://github.com/hyperledger/iroha/pull/1248/files#diff-b12959003054cb5d591512ab91741f93R19